content/pages/sprint/2025/02-hsinchu.rst

```diff
@@ -56,6 +56,20 @@ development insights in this event.
 We would like to provide a supportive and friendly environment for all attendees to support more developers
 to join in the open-source communities. 
 
+Technical sharing
```

Comment:
Could you please indicate the room for the talk?

Reply:
I plan to use the largest room in NTHU CTC, but currently I can not obtain the room number.
I will record the room number on Feb. sprint.

Reply:
Sounds good.  And we should also add a schematic for the floor plan in the page next time.  Let us discuss how to make it happen with the community.